Geometry_Engine/Query/IsVertical.cs

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Geometry;
+using BH.oM.Reflection.Attributes;
+using System;
+using System.ComponentModel;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Evaluates whether a line is vertical from its start- and endpoints projected 2d-distance")]
+        [Input("line", "The line to determine the verticality of")]
+        [Output("isVertical", "If it is vertical")]
```

Comment:
Can this be improved? Not sure this is the most beneficial of output descriptions? @IsakNaslundBh thoughts?

Reply:
Good to link to our agreed definition of what vertical means - i.e. tolerances etc. on wiki perhaps

Reply:
If this can be fixed, Linking over to the Wiki, then I am Happy to approve and merge this PR

Reply:
Sure, but it is a bit strange with a link to the structure wiki in the geometry engine

Reply:
> Sure, but it is a bit strange with a link to the structure wiki in the geometry engine

Think that is fine, at least for now. BEnv will need to have those kinds of things defined for things like ducts etc at one point as well, and once that happens, we could move the wiki out of the structures convention page.